anonlink/solving.py

```diff
@@ -13,7 +13,9 @@
 
 
 def greedy_solve(
-    candidates: _typechecking.CandidatePairs
+    candidates: _typechecking.CandidatePairs,
+    *,
+    agreement_threshold: float = .5
```

Comment:
Please document `agreement_threshold`; include some guidance about the consequences of picking different values.

Reply:
Oh oops. I forgot to update the docstring…

Reply:
@unzvfu Could I ask you to have a look at the changes to the docstring? In particular, I'm curious whether the documentation is sufficient and whether the language are clear enough (I found the role of `agreement_threshold` difficult to phrase in the constraints of a docstring).

Reply:
Looks good. Just need to explicitly justify the choice of default parameter.